src/testkit/scala/tools/testkit/TempDir.scala

```diff
@@ -28,3 +31,11 @@ object TempDir {
     throw new IOException(s"Failed to create directory")
   }
 }
+
+case class ForDeletion(path: Path)
+object ForDeletion {
+  import scala.util.Using.Releasable
+  implicit val deleteOnRelease: Releasable[ForDeletion] = new Releasable[ForDeletion] {
+    override def release(releasee: ForDeletion) = if (!Properties.isWin) Files.delete(releasee.path)
```

Comment:
Add a comment about the `isWin` special case?

Reply:
Windows doesn't feel special.